vue-frontend/src/components/home-new/BlogSection.vue

```diff
@@ -0,0 +1,210 @@
+            v-if="
+              (blogs.length == 2 && index == 1) ||
+              (blogs.length == 3 && index == 2)
+            "
+            class="tw-hidden md:tw-block md:tw-ml-[4.375rem] tw-mt-[1.125rem] 2xl:tw-ml-[5.313rem] md:tw-w-[18.438rem] lg:tw-w-[27.5rem] 2xl:tw-w-[42.375rem] tw-h-[0.063rem] tw-bg-[#C6C6C6]"
+          />
+          <hr
+            v-else
+            class="tw-w-[17.5rem] md:tw-w-[18.438rem] lg:tw-w-[27.5rem] 2xl:tw-w-[42.375rem] tw-h-[0.063rem] tw-ml-[3.20rem] md:tw-ml-[4.375rem] 2xl:tw-ml-[5.313rem] tw-mt-[1.125rem] tw-bg-[#C6C6C6]"
+          />
+
+          <div
+            v-show="activeIndex === index"
+            class="tw-block md:!tw-hidden tw-ml-[3.125rem] tw-mt-[0.5rem] tw-mb-[0.5rem] tw-max-w-[17.625rem] tw-max-h-[10rem] tw-animate-fadeIn"
+          >
+            <a :href="blog.link">
+              <img
+                :src="blog.thumbnail"
+                class="tw-object-cover"
+                loading="lazy"
+                :alt="blog.title"
+              />
+            </a>
+            <hr
+              v-if="
+                (blogs.length == 2 && index == 0) ||
+                (blogs.length == 3 && index != 2)
+              "
+              class="md:tw-hidden tw-w-[17.5rem] md:tw-w-[18.438rem] lg:tw-w-[27.5rem] 2xl:tw-w-[42.375rem] tw-h-[0.063rem] tw-ml-[0.2rem] md:tw-ml-[4.375rem] 2xl:tw-ml-[5.313rem] tw-mt-[1.125rem] tw-bg-[#C6C6C6]"
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script type="module">
+import bgMobile from "@/assets/images/blog/bg/bg400.svg";
+import bg from "@/assets/images/blog/bg/bg2400.svg";
+import axios from "axios";
+import Config from "@/config.js";
+
+export default {
+  data() {
+    return {
+      width: 680,
+      bgMobile,
+      bg,
+      activeIndex: 0,
+      blogs: "",
+      blogsURL: Config.BLOG_URL,
+    };
+  },
+
+  mounted() {
+    this.getBlogs();
+    this.width = window.innerWidth;
+  },
+  methods: {
+    openLink() {
+      if (this.width < 680) {
+        return;
+      }
+      for (let i = 0; i < this.blogs.length; i++) {
+        window.open(this.blogs[i].link, "_blank");
+      }
+    },
+    getBlogs() {
+      axios
+        .get("https://api.rss2json.com/v1/api.json", {
+          params: {
+            rss_url: "https://medium.com/feed/canopas",
+          },
+        })
+        .then((response) => {
+          var blogs = response.data.items.filter(function (item) {
+            return !item.title.includes("Weekly");
+          });
+
+          for (let i = 0; i < blogs.length; i++) {
+            /** date in MMM DD, YYYY format */
+
+            blogs[i].pubDate = new Date(blogs[i].pubDate).toLocaleDateString(
+              "en-US",
+              {
+                month: "long",
+                day: "2-digit",
+              }
+            );
+            blogs[i].pubDate = blogs[i].pubDate.split(" ", 2);
+          }
+
+          blogs = blogs.slice(0, 3);
```

Comment:
Let's first slice blogs, then update the date. It will reduce the length of data and time of conversion